### PR TITLE
Add procedure to create API-only user

### PR DESCRIPTION
**Pull request by maximiliankolb**
You can create a user with a secret password that you don't save to disable log-in to the Foreman Web UI.

Cherry-pick into:

* [x] Foreman 3.2
* [x] Foreman 3.1

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Kindly requesting a review @melcorr

File: guides/common/modules/proc_creating-an-api-only-user.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[id="Creating_an_API_Only_User_{context}"]
+= Creating an API-Only User
+
+You can create users that can only interact with the {Project} API.
```
Comment (melcorr) [inline guides/common/modules/proc_creating-an-api-only-user.adoc:4]:
```suggestion
You can create users that can interact only with the {Project} API.
```

File: guides/common/modules/proc_creating-an-api-only-user.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+.Prerequisite
+. You have created a user and assigned roles to them.
+Note that this user must be authorized internally.
+For more information, see xref:Creating_a_User_{context}[] and xref:Assigning_Roles_to_a_User_{context}[].
+
+.Procedure
+. Log-in to your {Project} as admin.
```
Comment (melcorr) [inline guides/common/modules/proc_creating-an-api-only-user.adoc:12]:
```suggestion
. Log in to your {Project} as admin.
```
IBM Style pp 342

File: guides/common/modules/proc_creating-an-api-only-user.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+. Navigate to *Administer > Users* and select a user.
+. On the *User* tab, set a secret password.
```
Comment (melcorr) [inline guides/common/modules/proc_creating-an-api-only-user.adoc:14]:
```suggestion
. On the *User* tab, set a password.
```

File: guides/common/modules/proc_creating-an-api-only-user.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# openssl rand -hex 32
+----
+. On the *Personal Access Tokens* tab, click *Add Personal Access Token*.
+. Enter a name for the token.
+. Optional: Set an expiration date.
+. Click *Submit* to create the API token.
+. Copy the shown API token.
```
Comment (melcorr) [inline guides/common/modules/proc_creating-an-api-only-user.adoc:26]:
```suggestion
. Copy the API token.
```

**Review by melcorr — COMMENTED** [review]
Few tiny nits.

File: guides/common/modules/proc_creating-an-api-only-user.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-an-api-only-user.adoc]: Applied.

File: guides/common/modules/proc_creating-an-api-only-user.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-an-api-only-user.adoc]: Applied.

File: guides/common/modules/proc_creating-an-api-only-user.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-an-api-only-user.adoc]: Applied.

File: guides/common/modules/proc_creating-an-api-only-user.adoc (hunk 4 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-an-api-only-user.adoc]: Applied.

**Review by maximiliankolb — COMMENTED** [review]
Thanks for your review! I've addressed all issues and rebased to master.

**Pull request merged**